### ISSUE TITLE
Error in starting application

### ISSUE THREAD

**Issue by alepironi** [devonfw/devon4ng-application-template#159]
Trying to follow the tutorial here:
https://github.com/devonfw/cobigen/blob/master/documentation/howto-devonfw-ide-CobiGen-PoC-E2E.adoc

I got this error when I tried the command "ng serve -o"

![error](https://user-images.githubusercontent.com/55445310/94484502-d309cd80-01dc-11eb-81dc-3aa752b51d5c.jpg)

Actually the error happened on the laptop of Stefano Rossini; it worked when I did it on my laptop.

**Comment by sjimenez77** [comment]
Hi @alepironi,

I have tried to reach Stefano via Teams but he did not answered. 

**I do not consider this a bug** since we have been able to run the application with no issues in multiple computers, operative systems and different devonfw IDE installations and we could not replicate the issue. All the times it worked well as expected. 

I suggest to reinstall the devonfw IDE in a different disk location and try to clone and install the application template again. If it works I suggest to remove the previous devonfw IDE installation.
